app/views/rapidfire/question_groups/index.html.haml

```diff
@@ -1,3 +1,5 @@
+- content_for :after_title, ' - Question Groups'
```

Comment:
For consistency, I think we should move towards using the same kind of dash for all these. An en-dash ( `–` ) is probably the right one, and we use that on many of the existing titles. (We currently have a mix of en-dash and hyphen (`-`), but I think en-dash is what we should be using everywhere.)

Reply:
so am suppose to change all the titles i have added to use an en-dash right?